app/main.jsx

```diff
@@ -12,19 +12,25 @@ import ProductContainer from './containers/ProductContainer'
 // import WhoAmI from './components/WhoAmI'
 import App from './components/App'
 
-import {fetchProducts} from './reducers/products'
+import {fetchProducts, fetchProductById} from './reducers/products'
 
 function onProductsEnter() {
   store.dispatch(fetchProducts())
 }
 
+// do we really need to go fetch this info again? Maybe to get reviews....
```

Comment:
Hmmm... yeah, it makes sense to me to do this if we assume that someone might leave the page open for  a long time.... Let's stick to this and double check with Joe when we have review on Monday.
I don't see this related to passing info from one view to another. Product Details view can get the needed info from the state without having to go to the DB.